app/content/css/styles.css

```diff
@@ -111,6 +111,8 @@
   position: absolute;
   left: calc(50% - 250px);
   top: calc(50% - 119px);
+  display: inline-block; 
+  z-index: 1000;
```

Comment:
I would suggest to have some conventions about z-index levels used in the styles. It will be easier to support such styles in the future if the code will follow such conventions